include/oneapi/dpl/pstl/hetero/dpcpp/unseq_backend_sycl.h

```diff
@@ -31,9 +31,9 @@ namespace unseq_backend
 {
 // helpers to encapsulate void and other types
 template <typename _Tp>
-using void_type = typename ::std::enable_if<::std::is_void<_Tp>::value, _Tp>::type;
+using void_type = ::std::enable_if_t<::std::is_void<_Tp>::value, _Tp>;
```

Comment:
I am wondering why this name is not uglified and also why is the name so bad? `__enable_if_void` would be much better.

As I can see, they are both `void_type` and `non_void_type` are unused. If it's the case it's better to remove those at all.

Reply:
Fixed: `void_type` and `non_void_type` removed